dev_env/core/runner.py

```diff
@@ -0,0 +1,105 @@
+import json
+import os
+import re
+import shutil
+import time
+
+from core.prompts import Prompt, Colors
+from core.utils import Runner
+
+
+class DockerCompose(Runner):
+    REQUIRED_PORTS = [8000]
+
+    def generate_config(self):
+        config = {
+            "admins": [],
```

Comment:
```suggestion
            "admins": [],
            "allowed_hosts": [],
```
allowed_hosts too

Reply:
Allowed hosts will not be changable on local environment.  It's hardcoded to be localhost.